### DISCUSSION TITLE
Timed out waiting for response. Try enabling debug mode to see more information.

### DISCUSSION THREAD

**Discussion by beiwushijie** [waylaidwanderer/node-chatgpt-api#391] [Help]
When I asked newbing to output a long content, he would get stuck somewhere after outputting a lot of content, and then this error occurred and all the previous output content was cleared. Is there any way to adjust this time limit? , or other solutions. Error content: Timed out waiting for response. Try enabling debug mode to see more information.
<img width="902" alt="截屏2023-05-29 10 12 51" src="https://github.com/waylaidwanderer/node-chatgpt-api/assets/61958116/0e156f4b-b12d-41ed-9f43-1fe1d2caa8aa">

**Comment by waylaidwanderer** [comment]
Is there a reason why you closed this?

**Comment by beiwushijie** [comment]
Because no one answered my question, I thought people might not be interested, so I closed it




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年5月29日(星期一) 下午5:40
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; "State ***@***.***&gt;; 
主题: Re: [waylaidwanderer/node-chatgpt-api] Timed out waiting for response. Try enabling debug mode to see more information. (Discussion #391)





 
Is there a reason why you closed this?

**Comment by waylaidwanderer** [comment]
I increased the timeout value from 180s to 300s in `@waylaidwanderer/chatgpt-api@1.36.4`. See if that helps.
